healthtools/scrapers/base_scraper.py

```diff
@@ -207,9 +230,20 @@ def delete_elasticsearch_docs(self):
             return response
         except Exception as err:
             if "NoSuchKey" in err:
-                print "ERROR - delete_elasticsearch_docs() - no delete file present"
+                requests.post(
+                    SLACK['url'],
+                    data=json.dumps(
+                        {"text": "```ERROR - delete_cloudsearch_docs() - no delete file present```"}),
+                    headers={'Content-Type': 'application/json'}
+                    )
+                print "ERROR - delete_cloudsearch_docs() - no delete file present"
                 return
-            print "ERROR - delete_elasticsearch_docs() - {} - {}".format(type(self).__name__, str(err))
```

Comment:
We should still maintain error logs being printed incase Slack is not preferred / used by other people who pick up this code. Also, dev environments might prefer not to send to Slack.